test/e2e/config/packet-ci.yaml

```diff
@@ -114,7 +114,7 @@ providers:
       # Add cluster templates
       - sourcePath: "../data/v1alpha3/cluster-template-packet-ccm.yaml"
       - sourcePath: "../data/v1alpha3/cluster-template-cpem.yaml"
-  - name: v0.5.0 #latest in the v1beta1 series
+  - name: v0.6.0 #latest in the v1beta1 series
     value: "../../../config/default"
     replacements:
     - old: "image: .*/cluster-api-provider-packet:.*"
```

Comment:
If updating `clusterctl-settings.json` for the version number, the entry below for `v1.1.99` would also need to be changed below.

Reply:
Went with having both in the file so we can try out the upgrade tests.